Sources/Testing/Running/Runner.swift

```diff
@@ -114,12 +114,55 @@ extension Runner {
     }
   }
 
+  /// Enumerate the elements of a sequence, parallelizing enumeration in a task
+  /// group if a given plan step has parallelization enabled.
+  ///
+  /// - Parameters:
+  ///   - sequence: The sequence to enumerate.
+  ///   - step: The plan step that controls parallelization. If `nil`, or if its
+  ///   ``Runner/Plan/Step/action`` property is not of case
+  ///   ``Runner/Plan/Action/run(options:)``, the
+  ///   ``Configuration/isParallelizationEnabled`` property of this runner's
+  ///   ``configuration`` property is used instead to determine if
+  ///   parallelization is enabled.
+  ///   - body: The function to invoke.
+  ///
+  /// - Throws: Whatever is thrown by `body`.
+  private func _forEach<E>(
```

Comment:
Would this be more readable as an instance method in an extension of `Sequence`?

Reply:
Possibly, although that would require hoisting it out and probably into a separate file SequenceAdditions.swift, whereas it's really an implementation detail of `Runner`.